.github/actions/write_to_db/action.yml

```
@@ -0,0 +1,62 @@
+name: Write data to InfluxDB
+description: Gather workflow and test data with `gather_data.py` and write it to the InfluxDB
+
+inputs:
+  influx_token:
+    required: true
+    description: InfluxDB API token
+  influx_org:
+    required: false
+    default: tarantool
+    description: InfluxDB account organisathion
+  influx_url:
+    required: true
+    description: InfluxDB URL
+  since:
+    required: false
+    default: 30d
+    description: set `--since` option for `gather_data.py`. Format `Nd|h`, 
+                  i.e. `30d`, `12h`
+  deployment_url:
+    required: false
+    description: Link to open on 'View deployment' button
+
+runs:
+  using: composite
+  steps:
+    - name: Get bucket base
+      run: |
+        if [[ "${{ github.event_name }}"=="pull_request" ]]; then
+          echo 'BUCKET_BASE<<EOF' >> $GITHUB_ENV
```

```suggestion
          echo 'ENV_NAME<<EOF' >> $GITHUB_ENV
```

this variable is used in bucket names, env URL,s and maybe somewhere else.

---

done